Source/error.cpp

```diff
@@ -20,19 +20,23 @@ namespace devilution {
 
 namespace {
 
-std::deque<std::string> DiabloMessages;
+struct MessageEntry {
+	std::string text;
+	uint32_t duration;  // Duration in milliseconds

```

Comment:
Please fix clang-format issue here

Reply:
```suggestion
	uint32_t duration; // Duration in milliseconds
```

Reply:
Message was unclear, but I think it's the double space